### PR TITLE
Internal: Add 'Free to Pro' promotion to the Video widget [ED-13399]

### PR DESCRIPTION
**Pull request by Barcov**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: includes/editor-templates/panel.php [hunk 1]
```diff
@@ -188,7 +188,7 @@
 
 	<?php if ( ! Utils::has_pro() ) : ?>
 	<# if ( elementData.upsale_data && elementData.upsale_data.description ) { #>
-	<div class="elementor-nerd-box">
+	<div class="elementor-nerd-box widget-upsale-promotion">
```
[the remark this answers is not in the document]
Comment (Barcov) [reply includes/editor-templates/panel.php:191]: Thanks Good suggestion.

**Review by davseve — DISMISSED** [review]

File: tests/playwright/sanity/video.test.ts [hunk 2]
```diff
@@ -27,6 +27,30 @@ test.describe( 'Video tests inside a container @video', () => {
...
+		const editor = new EditorPage( page, testInfo );
+		await wpAdmin.openNewPage();
+		await editor.closeNavigatorIfOpen();
+
+		const containerId = await editor.addElement( { elType: 'container' }, 'document' );
+		const videoId = await editor.addWidget( widgets.video, containerId );
+		const promoArea = await page.locator( '.elementor-nerd-box' );
+
+		// Act.
+		// Set container padding to 0.
```
Comment (davseve) [inline tests/playwright/sanity/video.test.ts:44]: @Barcov Where is the action that sets the padding to 0?

File: tests/playwright/sanity/video.test.ts [hunk 3]
```diff
@@ -27,6 +27,30 @@ test.describe( 'Video tests inside a container @video', () => {
...
+		const context = await browser.newContext();
+		const page = await context.newPage();
+		const wpAdmin = new WpAdminPage( page, testInfo );
+		const editor = new EditorPage( page, testInfo );
+		await wpAdmin.openNewPage();
+		await editor.closeNavigatorIfOpen();
+
+		const containerId = await editor.addElement( { elType: 'container' }, 'document' );
+		const videoId = await editor.addWidget( widgets.video, containerId );
+		const promoArea = await page.locator( '.elementor-nerd-box' );
```
Comment (louiswol94) [inline tests/playwright/sanity/video.test.ts:41]:
```suggestion
		const promoArea = await page.locator( '.elementor-nerd-box--upsale' );
```
maybe?

File: includes/widgets/video.php [hunk 4]
```diff
@@ -1377,4 +1377,12 @@ private function render_hosted_video() {
 		<video class="elementor-video" src="<?php echo esc_attr( $video_url ); ?>" <?php Utils::print_html_attributes( $video_params ); ?>></video>
 		<?php
 	}
+
+	protected function get_upsale_data() {
+		return [
+			'title' => __('Grab your visitors\' attention'),
```
Comment (davseve) [inline includes/widgets/video.php:1383]:
```suggestion
			'title' => __( 'Grab your visitors\' attention' ),
```

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**